guided-charts/aws-hyperpod/templates/authmiddleware/deployment.yaml

```diff
@@ -100,6 +100,12 @@ spec:
             value: "{{ .Values.authmiddleware.cookieSameSite }}"
           - name: PATH_REGEX_PATTERN
             value: "{{ .Values.authmiddleware.pathRegexPattern }}"
+          - name: ROUTING_MODE
+            value: "{{ .Values.authmiddleware.routingMode }}"
+          - name: WORKSPACE_NAME_SUBDOMAIN_REGEX
+            value: "{{ .Values.authmiddleware.workspaceNameSubdomainRegex }}"
+          - name: WORKSPACE_NAMESPACE_SUBDOMAIN_REGEX
```

Comment:
could you make the name consistent w/ `PATH_REGEX_PATTERN` ? as in, we should decide whether the noun we use is `_REGEX` or `_REGEX_PATTERN` and stick to it. Similarly, we should decide if we need `WORKSPACE_` prefix or not. I'm fine either way, but please keep it consistent.